pkg/velero/server/config.go

```diff
@@ -81,8 +81,10 @@ type ServerConfig struct {
 	// Backup all volumes with pod volume file system backup by default.
 	// +optional
 	DefaultVolumesToFsBackup *bool `json:"default-volumes-to-fs-backup,omitempty"`
-	// uploaderType, oadp only support restic at the moment
-
+	// The type of uploader to transfer the data of pod volumes, the supported values are 'restic', 'kopia' (default "restic")
+	// Commented out, please see:
+	//    https://github.com/openshift/oadp-operator/pull/1102#discussion_r1286054062
+	// UploaderType string `json:"uploader-type,omitempty"`
```

Comment:
Instead of commenting out, lets just remove these lines. I don't think we'll ever want to configure NodeAgent here.

Reply:
removed.